R/manual_layer_udf.R

```diff
@@ -37,6 +37,14 @@ execute_generic_udf <- function(namespace, udf=NULL, registered_udf_name=NULL, a
 
   # We rely on UDFLanguage$new to do enum-matching
   generic_udf$language <- UDFLanguage$new(language)
+  if (language == 'r') {
+    # For server-side stats; also for matching server-side Docker image version
+    # to client-side (us) version.
+    #
+    # TODO: How to specify lang vers for R calling registered Python UDFs.
+    # Probably make this a required argument (if language != 'r') once that's all deployed server-side.
+    generic_udf$version <- paste(R.version$major, R.version$minor, sep='.')
```

Comment:
This is a little maddening but there is also `R.version.string` (for me `"R version 4.1.2 (2021-11-01)"`) or `format(getRversion())` which is the function with the nice comparison object, but formatted to text for you here (`"4.1.2"` for me).